test/unit/serializer.cpp

```diff
@@ -160,19 +168,35 @@ struct serializer_test
     void
     testSyntax()
     {
-        serializer sr(1024);
+        context ctx;
+        serializer sr(ctx, 1024);
         response res;
 
         sr.start(res);
+        sr.reset();
+
         sr.start(res, buffers::const_buffer{});
+        sr.reset();
+
         sr.start(res, buffers::mutable_buffer{});
+        sr.reset();
+
         sr.start<test_source>(res, "12345");
+        sr.reset();
+
         sr.start(res, make_const(buffers::const_buffer{}));
```

Comment:
where is `make_const` from?

Reply:
It seems like it's from the actual test file itself and from the looks of it, it's trying to translate rvalue references into const lvalue ones. Not sure if we should keep it or remove it yet.

Reply:
if its local to the test we can leave it alone